messages/extend-config-missing.txt

```diff
@@ -0,0 +1,3 @@
+ESLint couldn't find the config "<%- configName %>" to extend from. Please, check the name of the config is correct.
+
+If you still can't figure out the problem, please stop by https://gitter.im/eslint/eslint to chat with the team.
```

Comment:
I know we have other error messages that also link to the chatroom, but have we considered using something other than a hardcoded URL? If we ever switch from Gitter to something else, anyone with an old ESLint version will get directed to an inactive support link. I wonder if we should link people to `eslint.org/chatroom` or something, and then have that URL redirect to Gitter.